heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -109,6 +118,28 @@ public boolean onRestart(Scheduler.RestartTopologyRequest request) {
     return controller.restartJob(containerId);
   }
 
+  @Override
+  public boolean onUpdate(Scheduler.UpdateTopologyRequest request) {
+    try {
+      updateTopologyManager.updateTopology(
```

Comment:
This is related to #1222 

Currently method `createAuroraProperties` provides instance distribution to containers. On topology update the instance distribution for existing containers, e.g. `C`, can change. However the aurora property map for container `C` will not change. As a result if container `C` restarts, it will use old instance plan. Will the executor ignore the old instance distribution?